src/golang/lib/airflow/client.go

```diff
@@ -39,15 +39,34 @@ func newClient(ctx context.Context, authConf auth.Config) (*client, error) {
 
 // getDagRuns returns all of the Airflow DAGRuns for the Airflow DAG specified.
 func (c *client) getDagRuns(dagId string) ([]airflow.DAGRun, error) {
-	dagRunsResp, resp, err := c.apiClient.DAGRunApi.GetDagRuns(
-		c.ctx,
-		dagId,
-	).OrderBy("start_date").Execute()
-	if err != nil {
-		return nil, wrapApiError(err, "GetDagRuns", resp)
+	limit := 100 // This is the max number of DAG runs that can be returned in each response.
```

Comment:
nit: consider `limitPerPage`. I thought this is the total limit until I read the rest of the code